test/TestCliRolePaths.py

```diff
@@ -82,3 +82,21 @@ def test_run_role_name_invalid(self):
 
         result = run_ansible_lint(cwd=cwd, role_path=role_path)
         assert '106 Role name invalid-name does not match' in str(result)
+
+
+def test_run_playbook_github():
+    """Call ansible-lint simulating GitHub Actions environment."""
+    cwd = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+    role_path = 'examples/example.yml'
+
+    env = {
+        "GITHUB_ACTIONS": "true",
+        "GITHUB_WORKFLOW": "foo"
+    }
```

Comment:
https://github.com/ansible/ansible-lint/pull/911#discussion_r460552675

Reply:
Updated code to assure these are added only when vars are present.

Reply:
This still needs negative tests.

Reply:
I does have them, I forgot to push the last version.